test/html-fragments.spec.tsx

```diff
@@ -79,4 +79,16 @@ describe('using a number attribute', () => {
 describe('custom elements', () => {
     testEqual('<custom-element a-custom-attr="value" custom-li-attr="li"></custom-element>', () => <customElement ACustomAttr="value" customLIAttr="li"></customElement>);
     testEqual('<div some-data="s"></div>', () => <div some-data="s"></div>);
+});
+
+describe("stateless functions", () => {
```

Comment:
Awesome! Thanks for adding a test. Where does the name `stateless functions` come from? Is it a react thing? Or did you think of it yourself.

The thing is, you *could* create a stateful function this way as well right? So i'm not sure "stateless" is the right name here.

Reply:
Hi, no problem to make the changes. Stateless functions was a half remembered thing from react (stateless functional components). I think it's fine to get rid of it.